### ISSUE TITLE
PnP-JS-Core SP.Utilities.Utility methods do not work

### ISSUE THREAD

**Issue by ismet** [koltyakov/sp-rest-proxy#19]
Sample code (sendEmail):

```
const emailProps = {
     To: ["ismet@spdomain.com"],
     Subject: "This email is about...",
     Body: "Here is the body. <b>It supports html</b>",
};

pnp.sp.utility.sendEmail(emailProps).then(_ => {
     console.log("Email Sent!");
});
```

Error:
`
POST http://localhost:8080/_api/contextinfo 404 (Not Found)
`

**Comment by koltyakov** [comment]
Hi @ismet,

What is the version of SharePoint and sp-rest-proxy you're facing the issue?

I've just checked on Proxy v.2.0.9 and SharePoint Online it worked for me. No errors in a console, in a browser and email on external gmail address was sent and received. 

Does the code work for you on a page in SharePoint?

Also, what is the version of PnP-JS-Core? In my case, I tested on CDN version of pnp-js-core v2.0.5.
